man/sd_journal_get_fd.xml

```diff
@@ -140,9 +140,10 @@ else {
     <function>sd_journal_get_events()</function>,
     <function>sd_journal_get_timeout()</function> and
     <function>sd_journal_process()</function> is
-    <function>sd_journal_wait()</function>. It will synchronously wait
-    until the journal gets changed. The maximum time this call sleeps
-    may be controlled with the <parameter>timeout_usec</parameter>
+    <function>sd_journal_wait()</function>. It will asynchronously
+    wait until the journal gets changed as the wake-up optimization
+    for notifications are introduced with latency. The maximum time
+    this call sleeps may be controlled with the <parameter>timeout_usec</parameter>
     parameter. Pass <constant>(uint64_t) -1</constant> to wait
```

Comment:
the latency is not added by sd_journal_wait(). it's already applied to sd_journal_get_events()/sd_journal_get_timeout().

Maybe leave the existing sentences in place but add an extra sentence to the end of the paragraph.

"Note that all of the above change notification interfaces will typically not notice change notifications instantly, but show some additional (though typically small) notification latency for multiple reasons: certain storage backends require time-based polling, in other cases wake-ups are optimized by coalescing events, in addition to OS-level IO/CPU scheduling latencies."

Or something like that. Could you rework this?